File: src/main/java/szewek/fl/proxy/FLProxy.java
package szewek.fl.proxy;

import szewek.fl.network.FLNetUtil;
import szewek.fl.network.FLNetUtilServer;
import szewek.fl.util.PreRegister;

public class FLProxy {
	public void addPreRegister(PreRegister pr) {}
	public void init() {}
	public FLNetUtil getNetUtil() {
		return FLNetUtilServer.THIS;
	}
}

File: src/main/java/szewek/fl/proxy/FLProxy.kt
package szewek.fl.proxy

import szewek.fl.network.FLNetUtil
import szewek.fl.network.FLNetUtilServer
import szewek.fl.util.PreRegister

open class FLProxy {
    open val netUtil: FLNetUtil
        get() = FLNetUtilServer.THIS

    open fun addPreRegister(pr: PreRegister) {}
    open fun init() {}
}
